queries/systemverilog/injections.scm

```diff
@@ -0,0 +1,2 @@
+((comment) @injection.content
+  (#set! injection.language "comment"))
```

Comment:
Maybe we should add
```scm
((macro_text) @injection.content
  (#set! injection.language "systemverilog"))
```
for macro definition highlights

Reply:
great idea